fuse_examples/imaging/classification/mnist/simple_mnist_starter.py

```diff
@@ -180,13 +180,20 @@ def run_train(params: dict) -> None:
         distributed=distributed,
     )
 
+    # A workaround to support multiple GPUs with a custom batch_sampler for both Lightning versions
+    #       see: https://lightning.ai/pages/releases/2.0.0/#sampler-replacement
+    kwargs = {}
```

Comment:
Shouldn't it be wrapped with if distributed?

Reply:
It could be wrapped, but it does nothing when it is not on distributed mode. (for example, the default value is `True`)

Reply:
I'll wrap it anyway for clarity, thanks!